src/components/WebLogView/Row.tsx

```diff
@@ -53,6 +54,11 @@ export function Row({ data, isSelected, onSelectRequest }: RowProps) {
           />
         </ResponseStatusBadge>
       </Table.Cell>
+      <Table.Cell>
+        <Text size="1">
+          {isNonStaticAssetResponse(data) ? 'Dynamic' : 'Static'}
```

Comment:
I was under the impression we wanted this to work similarly to the browser dev tools:
<img width="142" alt="image" src="https://github.com/user-attachments/assets/7120febb-bceb-4f82-84ea-e7a7dd55a2b0">
<img width="313" alt="image" src="https://github.com/user-attachments/assets/11197161-55db-4635-b3f5-3e9e100ae572">
If we only want to mark static requests, I suggest something that will take less horizontal space in `WebLogView` - maybe an icon or something like that